src/templates/learn.tsx

```diff
@@ -35,6 +36,10 @@ export default ({
         previous={previous}
         relativePath={relativePath}
       />
+      <TOCDesktop
+        heading="TABLE OF CONTENTS"
+        tableOfContents={tableOfContents}
+      />
```

Comment:
Check if there is TOC.

```suggestion
      {tableOfContents && (<TOCDesktop
        heading="TABLE OF CONTENTS"
        tableOfContents={tableOfContents}
      />)}
```

Reply:
Ahh, that's right.